src/main/java/org/opentripplanner/netex/loader/mapping/NetexMapper.java

```diff
@@ -79,19 +79,23 @@ public void mapNetexToOtp(NetexImportDataIndexReadOnlyView netexIndex) {
         FeedScopedIdFactory.setFeedId(agencyId);
         // Be careful, the order matter. For example a Route has a reference to Agency; Hence Agency must be mapped
         // before Route - if both entities are defined in the same file.
-        mapAgency(netexIndex);
+        mapOrganizations(netexIndex);
         mapStopPlaceAndQuays(netexIndex);
         mapRoute(netexIndex);
         mapTripPatterns(netexIndex);
         mapCalendarDayTypes(netexIndex);
         mapNoticeAssignments(netexIndex);
     }
 
-    private void mapAgency(NetexImportDataIndexReadOnlyView netexIndex) {
-        AgencyMapper agencyMapper = new AgencyMapper(netexIndex.getTimeZone());
+    private void mapOrganizations(NetexImportDataIndexReadOnlyView netexIndex) {
```

Comment:
The name says it maps Organizations (which are the parent type of Operators and Authorities) but the method seems to only actually map Authorities (no Operators).